dkms

```diff
@@ -1414,10 +1414,30 @@ clean_build()
     rm -rf "$dkms_tree/$module/$module_version/build"
 }
 
+sign_build()
+{
+    [[ -x "$(command -v kmodsign)" && -d "/var/lib/shim-signed/mok/" ]] || return 1
```

Comment:
returning 1, won't this cause failures for anyone missing these tools or directory structure?

Reply:
ok you are right, better just "return", but in the future we should really add a way to tell the user that something is missing...